tsl/src/process_utility.c

```diff
@@ -21,10 +22,24 @@ tsl_ddl_command_start(ProcessUtilityArgs *args)
 		DropdbStmt *stmt = castNode(DropdbStmt, args->parsetree);
 		remote_connection_cache_dropped_db_callback(stmt->dbname);
 	}
-
 	dist_ddl_start(args);
 }
 
+/* AlterTableCmds that need tsl side processing invoke this function
+ * we only process AddColumn command right now.
```

Comment:
It would be nice to improve the sentence grammatically to make it easier to read and have sentences syntactically correct, i.e., starting with a capital letter and a dot at the end. (I believe it is two sentences there)